.env

```diff
@@ -1,3 +1,4 @@
 AUTH0_SCOPE=openid profile
 
 DJANGO_BASE_URL=https://dtp-stat.ru
+DJANGO_CONTENT_FALLBACK=false
```

Comment:
I suggest us to keep this feature off by default. It helps detect what assets are missing during local dev. We also increase each other’s awareness of the feature by having to enable it by hand.

Reply:
I’ve set `DJANGO_CONTENT_FALLBACK` to `true` on Netlify.